R/wb_functions.R

```diff
@@ -40,46 +44,70 @@ dims_to_dataframe <- function(dims, fill = FALSE) {
     }
   }
 
-  # create data frame from rows/
-  dims_to_df(
-    rows = rows_out,
-    cols = cols_out,
-    fill = fill
-  )
+  if (is.null(filled)) {
+    # create data frame from rows/
+    dims_to_df(
+      rows = rows_out,
+      cols = cols_out,
+      fill = fill
+    )
+  } else {
+    # create data frame from rows/
+    dims_to_df2(
+      rows   = rows_out,
+      cols   = cols_out,
+      filled = filled,
+      fill   = fill
+    )
```

Comment:
The sort order is not necessarily correct. Should be fixed. Otherwise selecting first column name and first row name might provide incorrect results:
```R
> dims_to_dataframe("I2:J2;A1:B2;G5:H6")
     I    J    A    B    G    H
2 <NA> <NA> <NA> <NA>          
1           <NA> <NA>          
5                     <NA> <NA>
6                     <NA> <NA>
```